docs/wallets/spv.md

```diff
@@ -0,0 +1,60 @@
+
+- Light wallets depend on a remote service to watch owned addresses and provide notifications when transactions are received by the addresses. Notifications could be missed or incorrect.
+
+- Users will often be required to upload an extended public key to the central server so it is aware of all of owned addresses, potentially compromising the users privacy.
+
+- Light wallets do not validate the information they receive by checking the blockchain directly - they have to blindly trust the information provided by the central server.
+
+These concerns do not apply to SPV wallets because they connect directly to the decentralised peer-to-peer Decred network, they upload no private data to remote nodes, and they validate their own transactions by inspecting the blockchain directly.
+
+
+## Are there any disadvantages?
+
+- SPV does not support voting wallets. Voting wallets have the responsibility to vote on the validity of the last block, and a wallet cannot be sure of the validity unless it fully validates the whole blockchain leading up to that block. It is possible however to purchase tickets and allocate the voting rights to a [Voting Service Provider](../mining/how-to-stake.md#pos-using-a-voting-service-provider-vsp).
+
+- SPV wallets only download blocks which have transactions related to their owned addresses, which could potentially reveal more information about the wallet than if it downloaded every single block. This only presents a very minor decrease in privacy, but it is a decrease nonetheless. This is mitigated by downloading blocks from multiple peers so no single peer is able to see the full list of blocks downloaded by a wallet. Even if a passive observer on the network is able to see which blocks are downloaded by a wallet, they are not able to identify which transactions in those blocks are relevant.
+
+- Wallets operating in SPV mode are only able to validate the block headers they download and not the filters. This makes a "false-negative" attack possible: a malicious peer which knows a wallet is waiting for a particular transaction could send the wallet a fake filter which does not include the transaction, resulting in the wallet not downloading the block and so not becoming aware of the transactions existence. This transaction would still be visible to all fully validating nodes and wallets, and it will still appear in the [block explorer](../getting-started/using-the-block-explorer.md). One way to prevent this vulnerability is to add the hash of the filter into the block header, enabling SPV wallets to easily check the validity of the filters without having to download their blocks. A "false-positive" scenario is not possible - if a malicious node provides a fake filter which includes a non-existent transaction, the wallet will simply download the full block, compare it to the filter and discover that the filter is not genuine.
```

Comment:
It would be worth noting here that there are preliminary proposals to solving this issue by committing the filter to the part of the block header that is PoW-validated.